elastic/kibana
# [ResponseOps][Cases] Fix edit cases settings privilege

## Summary

Fixes https://github.com/elastic/kibana/issues/197650

Also fixes an issue where user has `cases: all ` and `edit case settings: false`, user was able to edit settings.

Used `permissions.settings` instead of `permissions.update` and `permissions.create` for custom fields and templates.

### How to test
- Verify by creating a user with different combinations of cases and edit case settings privileges

### Checklist
